api/user_actions.py

```diff
@@ -0,0 +1,35 @@
+"""Repeatable user actions involving Neo4J"""
+from neo4j.v1 import GraphDatabase
+
+DRIVER = GraphDatabase.driver(
```

Comment:
I know we decided to initialize this in many places for now, but I feel like there's got to be a way to do this in one place. Not a high priority, but something to think about now that we have a place for config, etc.

Reply:
I have a chore in the board to address this.